helm/blueapi/values.yaml

```diff
@@ -91,6 +91,14 @@ tracing:
       host: https://daq-services-jaeger # replace with central instance
       port: 4318
 
+
+logging:
+    graylog_export_enabled: True,
```

Comment:
Could: Disable it by default, requires a config change to beamlines but makes it log as-normal out-of-box, @DiamondJoseph what do you think?

Reply:
@callumforrester agreed, see my comment on the same line, I believe Dan has switched it to False by default but hasn't pushed?